src/stream.c

```diff
@@ -1595,7 +1626,6 @@ int stream_debug(struct re_printf *pf, const struct stream *s)
 	err |= re_hprintf(pf, " tx.enabled: %s\n",
 			  re_atomic_rlx(&s->tx.enabled) ? "yes" : "no");
 	err |= rtprecv_debug(pf, s->rx);
```

Comment:
Should we reorder `rtprecv_debug` and `rtp_debug`? So we can unlock earlier? Or is `tx.lock` needed for `rtprecv_debug` too?